README.md

```diff
@@ -0,0 +1,37 @@
+
+There is a new specification named "zaakgericht werken" (zgw) for case register systems. Zgw is part of the Common Ground specification for software used by Dutch municipalities (https://vng.nl/artikelen/common-ground). Zgw will replace an old specification called "zaak- en documentatieservices" (zds). The municipality of Súdwest Fryslân wants to migrate to the zgw standard without having to invest in zds interfaces. To this end, the municipality has built the system Open-ZaakBrug (brug means bridge). It acts as an interface between a zgw case register system on the one hand and zds systems connecting to the case register system on the other hand. Open-ZaakBrug is hosted at https://github.com/Sudwest-Fryslan/OpenZaakBrug. For a comparison of zgw and zds see https://vng.nl/sites/default/files/2022-03/20200703%20-%20Vergelijking%20ZGW%20APIs%20en%20ZDS_v1.0.pdf#:~:text=De%20Zaak-%20en%20Documentservices%20%28ZDS%29-standaard%20beschrijft%20hoe%20een,vakapplicatie%20en%20deze%20worden%20gekopieerd%20naar%20het%20zaaksysteem.
+
+This project, ZaakBrug 2.0, is meant to be a successor of Open-ZaakBrug. It leverages the Frank!Framework (https://wearefrank.nl/en/framework/) and its debugger Ladybug. The debugger shows for each incoming request which steps are executed to process the message. If there are errors, this information can be used for easy debugging. ZaakBrug 2.0 does not only connect zds systems with case register system OpenZaak (https://openzaak.org/en), but with any case register system that implements zgw.
+
+# Starting ZaakBrug 2.0
+
+### Development in Visual Studio Code
+
+When you are doing development work on ZaakBrug 2.0, you may want to boot it using WeAreFrank!'s Frank!Runner. When you use the Frank!Runner, you can boot ZaakBrug 2.0 from within your Integrated Development Environment (IDE); we support the two IDEs Visual Studio Code and Eclipse.
+
+To boot ZaakBrug 2.0 using the Frank!Runner, you need version control system Git. Choose or create some empty directory in which you can clone Git repositories. For the sake of the argument, we assume it to be `work`. Please do the following:
+
+1. Open a command prompt and change directory to `work`.
+1. Clone the Frank!Runner: `git clone https://github.com/ibissource/frank-runner`.
+1. Clone ZaakBrug 2.0: `git clone https://github.com/ibissource/open-zaakbrug`.
+1. Start the Frank!Runner with one of the boot scripts in the Frank!Runner checkout directory: `start.bat` or `restart.bat` for Windows or `start.sh` or `restart.sh` for Linux or Mac. This step lets the Frank!Runner download Ant, a build tool for Java applications.
+1. Configure your IDE as described in the Frank!Runner's documentation, see https://github.com/ibissource/frank-runner.
+1. If your IDE is Visual Studio Code, you should have a link to open ZaakBrug 2.0, see number 2 in the figure below. Use the menu option number 1 to get access to the link. Click the link to open ZaakBrug 2.0 using the Frank!Runner.
```

Comment:
You should mantion that this uses the Task Explorer add-on for VSCode.

Reply:
Gedaan.